app/views/shared/_statements_table.html.haml

```diff
@@ -0,0 +1,48 @@
+- if @statements.empty?
+  %p.notice= t(".no_statements")
+- else
+  %table.table.table-striped.table-hover
```

Comment:
I'm fine sharing this table, but I think the only difference is the admin view sees the Account column and the user view shows the facility 

Worth noting, right now all statement-able accounts (POs and CCs) are tied to a specific facility. This might change in the future, so I don't think anything should change because of it.